src/test/java/com/bobocode/hoverla/bring/ApplicationContextBuilderTest.java

```diff
@@ -0,0 +1,78 @@
+package com.bobocode.hoverla.bring;
+
+import ch.qos.logback.classic.Level;
+import ch.qos.logback.classic.Logger;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestInstance;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+import org.junit.jupiter.params.provider.NullAndEmptySource;
+
+import java.util.stream.Stream;
+
+import static com.bobocode.hoverla.bring.BringApplication.getContextBuilder;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatNoException;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+class ApplicationContextBuilderTest {
+
+    private final String PACKAGE = "com.bobocode.hoverla.bring";
+    private final String EXCEPTION_MESSAGE = "Argument [packagesToScan] must contain at least one not null and not empty element";
+
+    @ParameterizedTest(name = "Throws exception when null or empty package name was provided")
+    @NullAndEmptySource
+    void throwsExceptionWhenNullOrEmptyPackages(String packageName) {
+        BringApplication.ApplicationContextBuilder builder = getContextBuilder()
+                .packagesToScan(packageName);
+        assertThatThrownBy(builder::build)
+                .isInstanceOf(IllegalArgumentException.class)
+                .hasMessageContaining(EXCEPTION_MESSAGE);
+    }
+
+    @DisplayName("Throws exception when no packages were provided")
+    @Test
+    void throwsExceptionWhenNoPackagesToScan() {
+        BringApplication.ApplicationContextBuilder builder = getContextBuilder();
+        assertThatThrownBy(builder::build)
```

Comment:
add an empty line between builder creation and assertions to improve readability

Reply:
done